### ISSUE TITLE
How to pass the args to the executable?

### ISSUE THREAD

**Issue by gzrjzcx** [Unity-Technologies/ml-agents#2174]
Hi there, is't possible to pass arguments to the executable so that it is able to change some thing at runtime(training process)?
  
For example, after building an executable for Linux platform, I know we are able to get the args from command line by using the `system.environment.GetCommandLineArgs()` API in C# script. But can we modify them within the training process? In terms of the `mlagents` tool, the command `mlagents-learn --config --env --run-id --train` is used for invoking the training process, how can we pass the args through this command and utilize the args within the training process? 
  
By the way, is't possible to modify the training hyperparameters like `learning rate` etc. listed in the config file? So that we can just use a script to submit all of the training tasks with different config settings for the server. I know we can edit config file by cmd directly, just curious if it is possible to pass args to config file.

**Comment by xiaomaogy** [comment]
Hi @gzrjzcx you can totally do these yourself by customizing our python code. Currently we don't have a clean way to provide these features.

**Comment by gzrjzcx** [comment]
@xiaomaogy Got it. Thanks.
